Sources/TokamakCore/Views/List.swift

```diff
@@ -0,0 +1,404 @@
+}
+
+extension List where SelectionValue == Never {
+  public init(@ViewBuilder content: () -> Content) {
+    selection = .one(nil)
+    self.content = content()
+  }
+
+  public init<Data, RowContent>(_ data: Data,
+                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == ForEach<Data, Data.Element.ID, HStack<RowContent>>,
+    Data: RandomAccessCollection, RowContent: View, Data.Element: Identifiable {
+    selection = .one(nil)
+    content = ForEach(data) { row in
+      HStack {
+        rowContent(row)
+      }
+    }
+  }
+
+  public init<Data, RowContent>(_ data: Data,
+                                children: KeyPath<Data.Element, Data?>,
+                                @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == OutlineGroup<Data,
+                                  Data.Element.ID,
+                                  HStack<RowContent>,
+                                  HStack<RowContent>,
+                                  DisclosureGroup<HStack<RowContent>, OutlineSubgroupChildren>>,
+    Data: RandomAccessCollection, RowContent: View, Data.Element: Identifiable {
+    self.init {
+      OutlineGroup(data, children: children) { row in
+        HStack {
+          rowContent(row)
+        }
+      }
+    }
+  }
+
+  public init<Data, ID, RowContent>(_ data: Data,
+                                    id: KeyPath<Data.Element, ID>,
+                                    children: KeyPath<Data.Element, Data?>,
+                                    @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == OutlineGroup<Data,
+                                  ID,
+                                  HStack<RowContent>,
+                                  HStack<RowContent>,
+                                  DisclosureGroup<HStack<RowContent>, OutlineSubgroupChildren>>,
+    Data: RandomAccessCollection, ID: Hashable, RowContent: View {
+    self.init {
+      OutlineGroup(data, id: id, children: children) { row in
+        HStack {
+          rowContent(row)
+        }
+      }
+    }
+  }
+
+  public init<Data, ID, RowContent>(_ data: Data,
+                                    id: KeyPath<Data.Element, ID>,
+                                    @ViewBuilder rowContent: @escaping (Data.Element) -> RowContent)
+    where Content == ForEach<Data, ID, HStack<RowContent>>,
+    Data: RandomAccessCollection, ID: Hashable, RowContent: View {
+    selection = .one(nil)
+    content = ForEach(data, id: id) { row in
+      HStack {
+        rowContent(row)
+      }
+    }
+  }
+
+  public init<RowContent>(_ data: Range<Int>,
+                          @ViewBuilder rowContent: @escaping (Int) -> RowContent)
+    where Content == ForEach<Range<Int>, Int, HStack<RowContent>>, RowContent: View {
+    selection = .one(nil)
+    content = ForEach(data) { row in
+      HStack {
+        rowContent(row)
+      }
+    }
+  }
+}
```

Comment:
<!--
  1 failure:  File should conta...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/List.swift;
  Line: 404;
-->
- :no_entry_sign: File should contain 400 lines or less: currently contains 404 (`file_length`)